src/ovp_pipeline/commands/link_suggest.py

```
@@ -0,0 +1,355 @@
+    *,
+    candidates_per_page: int,
+    suggestions_per_page: int,
+) -> list[dict[str, Any]]:
+    title = page.get("title") or page["slug"]
+    body = page.get("body") or ""
+    query_text = f"{title}\n{body[:800]}".strip()
+    if not query_text:
+        return []
+    try:
+        bm25 = search_knowledge_index(layout.vault_dir, query_text, limit=candidates_per_page)
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)